v2/jdbc-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/JdbcToBigQueryOptions.java

```diff
@@ -113,7 +125,8 @@ public interface JdbcToBigQueryOptions
       order = 8,
       groupName = "Target",
       description = "BigQuery output table",
-      helpText = "The BigQuery output table location.",
+      helpText =
+          "The BigQuery output table location, in the format <PROJECT_ID>:<DATASET_NAME>.<TABLE_NAME>.",
```

Comment:
Here we have format in the description and also example, it may be duplicate.
Should we remove "in the format <PROJECT_ID>:<DATASET_NAME>.<TABLE_NAME>" and follow that standard in example like "<PROJECT_ID>:<DATASET_NAME>.<TABLE_NAME>" ?

Reply:
Yeah, lets remove `in the format <PROJECT_ID>:<DATASET_NAME>.<TABLE_NAME>`